dev/CommandBarFlyout/CommandBarFlyout.xaml

```diff
@@ -323,9 +323,8 @@
                         <Grid x:Name="OuterContentRoot"
                             VerticalAlignment="Top"
                             Margin="{TemplateBinding Padding}"
-                            Width="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=FlyoutTemplateSettings.CurrentWidth}"
+                            MinWidth="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=FlyoutTemplateSettings.CurrentWidth}"
```

Comment:
Why does this solve the issue?

Reply:
The issue was that when we close the flyout, we immediately set the outer content root width to the closed width. That results in the button to immediately jump which then results in the different closing animations. By setting the MinWidth instead, the primary commandbar will not immediately shrink allowing the animation for the primary items to be the same as the animation for the secondary commandbar items flyout.